strings/tips.txt

```diff
@@ -0,0 +1,205 @@
+Where the space map levels connect is randomized every round, but are otherwise kept consistent within rounds. Remember that they are not necessarily bidirectional!
+You can catch thrown items by toggling on your throw mode with an empty hand active.
+To crack the safe in the vault, use a stethoscope or thermal drill.
+You can climb onto a table by dragging yourself onto one. This takes some time. Clicking on a table that someone else is climbing onto will knock them down.
+You can drag other players onto yourself to open the strip menu, letting you remove their equipment or force them to wear something. Note that exosuits or helmets will block your access to the clothing beneath them, and that certain items take longer to strip or put on than others.
+Clicking on a windoor rather then bumping into it will keep it open, you can click it again to close it.
+You can spray a fire extinguisher, throw items or fire a gun while floating through space to change your direction. Simply fire opposite to where you want to go.
```

Comment:
You use the serial/Oxford comma elsewhere, so I'd use it here:

You can spray a fire extinguisher, throw items**,** or fire a gun

"Opposite of" is also probably more idiomatic, or maybe even "fire in the opposite direction of"

Reply:
Fixed in next commit